documentation/docs/30-advanced/67-shallow-routing.md

```diff
@@ -59,7 +59,7 @@ For this to work, you need to load the data that the `+page.svelte` expects. A c
 		href="/photos/{thumbnail.id}"
 		on:click={async (e) => {
 			// bail if opening a new tab, or we're on too small a screen
```

Comment:
```suggestion
			// bail if opening a new tab, or we're on too small a screen
			// macOS uses metaKey. windows and linux use ctrlKey
```

Reply:
Since Chrome uses the 'open link in new tab' context menu text, how about the following?

```diff
- for clicking to open a new tab,
+ for opening a link in a new tab,
+ to open the clicked link in a new tab,
``

Reply:
I tweaked the suggestion a bit. what do you think about it now?

Reply:
That is intuitive. Can I add 'the' in-front of the key names?

> macOS uses the metaKey while windows and linux use the ctrlKey

Following citation is from Wikipedia.

> Similarly to the [Shift key](https://en.wikipedia.org/wiki/Shift_key), the Control key rarely performs any function when pressed by itself. The Control key is located on or near the bottom left side of most keyboards (in accordance with the international standard [ISO/IEC 9995-2](https://en.wikipedia.org/wiki/ISO/IEC_9995#ISO/IEC_9995-2)), with many featuring an additional one at the bottom right.

Reply:
> Can I add 'the' in-front of the key names?

It does read nicely with 'the', but I was kind of trying to keep it on one line. We might have to break it into two lines if it gets too long, so I don't know if it'd be worth it or not

Reply:
I personally think splitting each checks is more readable.

```js
// bail if the user wants to open the link in a new tab
// macOS uses the metaKey, windows/linux use the ctrlKey
if (e.metaKey || e.ctrlKey) return;

// bail if the user wants to open the link in a new window
if (e.shiftKey) return;

// bail if the screen is too narrow/small
if (innerWidth < 640) return;
```

It might not be the most concise code, but there are more checks to add.

Will be adding this commit for now (also since `e.metaKey` has to be reverted)